### ISSUE TITLE
Content Edits for 76960 (#2 of 2)

### ISSUE THREAD

**Issue by jknipes** [department-of-veterans-affairs/va.gov-team#88512]
## Issue Description

- Per [this Slack thread](https://dsva.slack.com/archives/CL181NRJQ/p1720545050246189), We still need to create a page to explain how to setup and conduct tests in staging. Also need to determine the implications of changing passwords when we're all using the same accounts.

- Continuation of the work begun in [ticket 85512](https://github.com/department-of-veterans-affairs/va.gov-team/issues/85512)

---
## Tasks
- [x] Review Slack Thread
- [x] Contact Naomi Marcussen to continue this work
- [x] Update website according to suggestions provided by customer  
- [x] Prepare page for deployment

## SMEs for this work:

- @naomimarcussen 
- Clint Little [tentative]

## Artifacts:
[Draft 85512 (76960): Testing in a live environment](https://vfs.atlassian.net/wiki/spaces/PWD/pages/3281354766?atlOrigin=eyJpIjoiNGJlYTE5ZmVkNmUxNDlhYjg2NjViYWZlMjU0Zjc2NmMiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ)
## Acceptance Criteria
- [X] (Conducting sessions) page will be updated to include the changes - SEE COMMENTS

Original support ticket [76960](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76960)

- Content should have its own page under [Conducting sessions](https://depo-platform-documentation.scrollhelp.site/research-design/research-conducting-sessions).

**Comment by jknipes** [comment]
Work is currently blocked by SME, looking to schedule into a future sprint.

**Comment by jknipes** [comment]
Hi @naomimarcussen, Thanks for all of your input, we are hoping to continue this work with you in a future sprint.

**Comment by jknipes** [comment]
10/29: waiting update from SME

**Comment by naomimarcussen** [comment]
We need to put this ticket on hold as we explore the possibility of Codespaces solving this problem.

**Comment by jknipes** [comment]
Instead of adding the changes to Conducting Sessions, a new page "[Using test accounts for usability testing in staging](https://depo-platform-documentation.scrollhelp.site/research-design/using-test-accounts-for-usability-testing-in-stagi)" has been added to the [Planning](https://depo-platform-documentation.scrollhelp.site/research-design/planning-moderated-studies) section. 
 
While unplanned for this sprint, significant work was done in preparation - [Draft 85512 (76960): Testing in a live environment](https://vfs.atlassian.net/wiki/spaces/PWD/pages/3281354766?atlOrigin=eyJpIjoiNGJlYTE5ZmVkNmUxNDlhYjg2NjViYWZlMjU0Zjc2NmMiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ) - this update solves the original request, and we are closing this ticket as _complete_.
